File: src/v2/pyZOMBIA.io/src/server/Network/PacketIds.py
from Logger.Logger import logger

packets = {}
for i, packet in {
  "PACKET_ENTER_WORLD": 0,
  "PACKET_ENTITY_UPDATE": 1,
  "PACKET_INPUT": 2,
  "PACKET_PING": 3,
  "PACKET_RPC": 4
}.items():
  packets[packet] = i
  packets[i] = packet

File: src/game/Engine/Network/PacketIds.js
const PacketIds = {
  "PACKET_ENTITY_UPDATE": 0,
  "PACKET_PLAYER_COUNTER_UPDATE": 1,
  "PACKET_SET_WORLD_DIMENSIONS": 2,
  "PACKET_INPUT": 3,
  "PACKET_ENTER_WORLD": 4,
  "PACKET_PING": 7,
  "PACKET_RPC": 9
}

Object.keys(PacketIds).forEach(packet => {
  PacketIds[PacketIds[packet]] = packet;
});

export default PacketIds;
